src/mame/drivers/savquest.cpp

```diff
@@ -869,13 +950,20 @@ ROM_START( savquest )
 	ROM_LOAD( "vgabios.bin",   0x000000, 0x010000, CRC(a81423d6) SHA1(a099af621ce7fbaa55a2d9947d9f07e04f1b5fca) )
 
 	ROM_REGION( 0x080, "rtc", 0 )    /* default NVRAM */
-	ROM_LOAD( "savquest_ds12885.bin", 0x0000, 0x080, BAD_DUMP CRC(e9270019) SHA1(4d900ca317d93c915c80a9053528b741746f08a1) )
+	ROM_LOAD( "savquest_ds12885.bin", 0x0000, 0x080, CRC(e9270019) SHA1(4d900ca317d93c915c80a9053528b741746f08a1) )
 
 	DISK_REGION( "ide:0:hdd:image" )
 	DISK_IMAGE( "savquest", 0, SHA1(b7c8901172b66706a7ab5f5c91e6912855153fa9) )
+	DISK_REGION( "ide:1:cdrom:image" )
+	// doesn't mount rom when available probably due to IRQ conflicts
+	// DISK_IMAGE_READONLY( "hostinv", 0, SHA1(3cb86c62e80be98a717172b717f7276a0e5f6830) )
+	DISK_REGION( "ide2:0:hdd:image" ) // requires mapping to be detectable
+	DISK_IMAGE_READONLY( "savquest", 0, SHA1(b7c8901172b66706a7ab5f5c91e6912855153fa9) )
```

Comment:
Is `ide:0:hdd:image` used at all?
Alternatively if there's a method that makes the disk copy to be available from this space in r/o, without the double file buffer loading.

Reply:
And why did you add the calchase comment? Completely different game

Reply:
If there is no rom mapping then the bios does not detect the drive at all. I tried a few other disks, was hoping to read the quake drive but it doesn't seem to be readable at all. The calchase drive actually has some utilities on it and a norton commander clone which was useful in exploring the installation. Will remove them again and and leave the secondary master unmapped.

Reply:
Removed at 033fb3a0